src/seedu/addressbook/Main.java

```diff
@@ -79,15 +80,23 @@ private void exit() {
     /** Reads the user command and executes it, until the user issues the exit command.  */
     private void runCommandLoopUntilExitCommand() {
         Command command;
-        do {
-            String userCommandText = ui.getUserCommand();
-            command = new Parser().parseCommand(userCommandText);
-            CommandResult result = executeCommand(command);
-            recordResult(result);
-            ui.showResultToUser(result);
-
-        } while (!ExitCommand.isExit(command));
-    }
+		do {
+			try {
+				File storageFile = new File("DEFAULT_STORAGE_FILEPATH");
+				if (!storageFile.isFile()) {
```

Comment:
can tell that you try to make the usage of exception more convincing! Good job! However, you can further refactor this method. ;)